apps/server/src/modules/board/repo/recursive-delete.visitor.spec.ts

```diff
@@ -173,6 +176,22 @@ describe(RecursiveDeleteVisitor.name, () => {
 		});
 	});
 
+	describe('visitDrawingElementAsync', () => {
```

Comment:
it should trigger deletion of tldraw data via adapter

Reply:
This test is calling adapter to delete tldraw data. Also adapter has its own test.

Reply:
I have added test should trigger deletion of tldraw data via adapter